main/acle.md

```diff
@@ -1888,6 +1889,146 @@ or compiler-generated functions.
 
 This attribute does not apply to AArch64.
 
+## Function Multi Versioning
+
+The specification for Function Multi Versioning is in [**Beta** state](#current-status-and-anticipated-changes)
+and may change or be extended in the future.
+
+Function Multi Versioning provides a convenient way to select the most
+appropriate version of a function at runtime. All versions of the function may
+be in the final binary. The compiler generates all supported versions and the
+runtime makes the selection at load time.
+
+The following attributes trigger the multi version code generation:
+`__attribute__((target_version("name")))` and `__attribute__((target_clones("name",...)))`.
+
+1. These attributes can't be mixed with each other.
+2. The `"default"` version means the version of the function that would be generated
+without these attributes.
+3. `"name"` is the dependent features from the tables below.
+4. The dependent features could be joined by the `+` sign.
+5. None of these attributes will enable the corresponding ACLE feature(s) associated
+to the `"name"` expressed in the attribute.
+6. These attributes have no effect on the calling convention, all versions must
+use the same calling convention.
```

Comment:
This can be confusing, as there are attributes that can modify the calling convention [1] of a specific function without being applied to all version. I would state is as follows:

```suggestion
6. These attributes have no effect on the calling convention.
```

[1] See `aarch64_vector_pcs` at https://github.com/ARM-software/abi-aa/blob/main/vfabia64/vfabia64.rst#vector-procedure-call-standard

Reply:
I'd like to keep the second part as a separate item. Since the callee calls all versions in the same way changing only one  calling convention one of the function won't work.